.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature
+about: Use this template to suggest a feature
+title: "[FEATURE NAME]"
+labels: feature
+---
+
+
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Context
+<!--- Why is this change important to you? How would you use it? -->
+<!--- How can it benefit other users? -->
+
```

Comment:
A section for adding screenshots `Screenshots of the issue`

Reply:
Yup we can do that. Great idea. Let me quickly add that.

Reply:
Now that I think of it, I think it is more suitable for PR template. What do you think?

Reply:
Well, I've seen screenshots on both issue and PR templates. So it might be a good idea to keep it.